### PR TITLE
Document how to install prerequisites in examples READMEs

### LINKED ISSUE #1987: Extend helloworld RST with pip install-dependency command
### Description

Provide instructions so a novice can start the Connexion v3 "helloworld" example on the first try.

### Expected behaviour

The example runs on the first try.

### Actual behaviour

It takes a few tries to install all the prerequisites.

### Steps to reproduce

`python hello.py`

### Proposed addition to `README.rst`

* Create a new virtual environment
* Run `pip install 'connexion[flask,swagger-ui,uvicorn]>=3.1.0'`

### Discussion :)

I realize this is fairly trivial. Just trying to save the next person a little time. 

And  if you don't mind another suggestion, you might consider renaming the directory to "helloworld_flask", to keep it parallel with "helloworld_async" and similar names.

Please comment. I don't want to waste your time with a PR if this looks silly to you.

### PR DESCRIPTION
**Pull request by chrisinmtown**
Add server url /openapi to JWT example
Add TODO to SQLAlchemy example, does not work with current versions
Correct name of module in v3 instructions
    
Fixes #1987

### CODE REVIEW

**Comment by chrisinmtown** [comment]
Please review, if you like this approach I will extend to other README.rst files.

**Comment by RobbeSneyders** [comment]
Thanks @chrisinmtown, this looks good to me. Would be great to extend it to the other examples.

**Comment by chrisinmtown** [comment]
I am extending the changes to all the examples. Please check example `sqlalchemy` which has become a bit stale. I dropped the `convert_unicode` parameter in the `create_engine` function and then it started, but the first GET attempt yields this:
```
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 123145410666496 and this is thread id 140704480263680.
```
Do you have time for this?

**Comment by RobbeSneyders** [comment]
Let's leave that example as is for now then. Would be great if you could add a notice that it is outdated to the README.

For the other examples, the `flask` extra is not needed for most of them.

**Comment by chrisinmtown** [comment]
> Let's leave that example as is for now then. Would be great if you could add a notice that it is outdated to the README.

I will ask the cloud a few questions and see if I can figure it out :)

> For the other examples, the `flask` extra is not needed for most of them.

Right, I copy-pasted a little too quickly, will adjust.

**Comment by chrisinmtown** [comment]
This PR has primarily documentation changes. I tweaked one API spec and one requirements file to make the examples work. Please consider.

**Comment by chrisinmtown** [comment]
Would you please give me an opinion here? It's a real challenge to contribute when people go silent for a full week at a time.

File: examples/jwt/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
-..[swagger-ui]
 python-jose[cryptography]
 Flask>=0.10.1
```
Comment (RobbeSneyders) [inline examples/jwt/requirements.txt:2]:
```suggestion
```

File: examples/sqlalchemy/README.rst [hunk 2]
```diff
@@ -2,14 +2,36 @@
...
+
+Create a new virtual environment and install the required libraries
+with these commands:
 
 .. code-block:: bash
 
+    $ python -m venv my-venv
+    $ source my-venv/bin/activate
+    $ pip install 'connexion[flask,swagger-ui,uvicorn]>=3.1.0'
     $ pip install -r requirements.txt
```
Comment (RobbeSneyders) [inline examples/sqlalchemy/README.rst:23]: We should remove the flask and connexio dependencies from `requirements.txt` if we install them directly on the command line.

**Review by RobbeSneyders — COMMENTED** [review]
Sorry for the delay @chrisinmtown 

Two small comments, then it's good to go for me.

File: examples/jwt/requirements.txt (hunk 1 above)
Comment (chrisinmtown) [reply examples/jwt/requirements.txt:2]: Ok, dropped the Flask line

File: examples/sqlalchemy/README.rst (hunk 2 above)
Comment (chrisinmtown) [reply examples/sqlalchemy/README.rst:23]: Ok, dropped the connexion and Flask lines from requirements.txt

**Review by RobbeSneyders — APPROVED** [review]
Thanks @chrisinmtown!

**Pull request merged**